app/_gateway_entities/group.md

```diff
@@ -21,17 +24,50 @@ api_specs:
 
 tools:
   - admin-api
-  - konnect-api
-  - kic
-  - deck
-  - terraform
 
 schema:
   api: gateway/admin-ee
   path: /schemas/Group
 
+faqs:
+  - q: What authentication types work with the Group entity?
+    a: |
+      The Group entity works with the following authentication protocols: [Basic authentication](/plugins/basic-auth/), [LDAP authentication](/plugins/ldap-auth-advanced/), and [OpenID Connect (OIDC)](/plugins/openid-connect/). Configuring an auth protocol to work with {{site.base_gateway}} and Kong Manager is done using `kong.conf`, for more information review our guide on [Configuring LDAP with Kong Manager](/how-to/configure-ldap-with-kong-manager).
 ---
 
-@todo
+## What is a Group?
+
+
+In {{site.base_gateway}}, the Group entity functions as a resource for [RBAC](/gateway/entities/rbac/#role-configuration). {{site.base_gateway}} Admins can map Permissions and Roles to a Group, and use the Group to simplify Role assignment across the {{site.base_gateway}} environment. 
+
+The Group resource can also be used to integrate identity providers like Okta with Kong Manager, letting you map relationships between your service directory mappings and Kong Manager Roles.
+
+
+## Service directory mapping
+
+With service directory mapping, Groups can be mapped to [RBAC Roles](/gateway/entities/rbac/#role-configuration/). When a user logs in to Kong Manager, they are identified with their Admin username and authenticated with user credentials from a service directory, like [LDAP](/plugins/ldap-auth/). The service directory creates a relationship with the associated RBAC Roles that are defined in {{site.base_gateway}}. 
```

Comment:
```suggestion
With service directory mapping, Groups can be mapped to [RBAC Roles](/gateway/entities/rbac/#role-configuration). When a user logs in to Kong Manager, they are identified with their Admin username and authenticated with user credentials from a service directory, like [LDAP](/plugins/ldap-auth/). The service directory creates a relationship with the associated RBAC Roles that are defined in {{site.base_gateway}}. 
```
Not this one, since it's an anchor.